src/app/calc.css

```
@@ -0,0 +1,26 @@
+#head{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.Calc {
```

the general convention is to use lowercase for css clases

```suggestion
.calc {
```

---

Sure thing!